ext/RMagick/rmenum.c

```diff
@@ -361,6 +361,38 @@ Enum_type_values(VALUE class)
     return rv;
 }
 
+/**
+ * Find enum object for the specified value.
+ *
+ * No Ruby usage (internal function)
+ *
+ * @param class the class type
+ * @param value the value for enum
+ * @return a enumerator
+ */
+
+VALUE
+Enum_find(VALUE class, int val)
+{
+    VALUE enumerators;
+    MagickEnum *magick_enum;
+    int x;
+
+    enumerators = rb_cv_get(class, ENUMERATORS_CLASS_VAR);
+    enumerators = rm_check_ary_type(enumerators);
+
+    for (x = 0; x < RARRAY_LEN(enumerators); x++)
+    {
+       VALUE e = rb_ary_entry(enumerators, x);
```

Comment:
Maybe you could use a more descriptive name here instead of just `e`?

Reply:
renamed :)

Reply:
Thanks!